sphinxcontrib/images.py

```diff
@@ -45,6 +45,7 @@
     requests_kwargs={},
     cache_path='_images',
     override_image_directive=False,
+    caption=False,
```

Comment:
I think ``show_caption`` would be better as it tells more more how to use it and it informs it's not a caption, it's switch. what do you think?